src/shared/bootspec.c

```diff
@@ -1288,7 +1288,11 @@ BootEntry* boot_config_find_entry(BootConfig *config, const char *id) {
 static void boot_entry_file_list(const char *field, const char *root, const char *p, int *ret_status) {
         int status = boot_entry_file_check(root, p);
 
-        printf("%13s%s ", strempty(field), field ? ":" : " ");
+        if (field)
+                printf("%13s: %s\n", field, root);
+
+        printf("%13s  %s", "", special_glyph(SPECIAL_GLYPH_TREE_RIGHT));
```

Comment:
In the light of 84be9b63ebac1d320b1e39ccf6e0895e621af887 we shouldn't just use `└─` here, but use `├─` for all but the last item we display...

The code for this should really be simple to make that work, because in the STRV_FOREACH() loops that call this function it's super easy to figure out if we are at the last entry or not...

(might be worse adding a new helper function that displays the whole strv btw, instead of duplicating the STRV_FOREACH loops)